vault/resource_consul_secret_backend_test.go

```diff
@@ -211,13 +225,16 @@ resource "vault_consul_secret_backend" "test" {
 }`, path, token)
 }
 
-func testConsulSecretBackend_bootstrapConfig(path, addr string) string {
+func testConsulSecretBackend_bootstrapConfig(path, addr, token, bootstrap string) string {
 	return fmt.Sprintf(`
 resource "vault_consul_secret_backend" "test" {
   path = "%s"
   description = "test description"
   address = "%s"
-}`, path, addr)
+  token = "%s"
+  bootstrap = "%s"
```

Comment:
`bootstrap` is a boolean, so it shouldn't be quoted

```suggestion
  bootstrap = %t 
```